src/views/ProductInventory.vue

```diff
@@ -0,0 +1,346 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-back-button default-href="/" slot="start" />
+        <ion-title>{{ $t("Product inventory") }}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button>
+            <ion-icon :icon="sync" slot="icon-only" />
+          </ion-button>
+          <ion-button>
+            <ion-icon :icon="downloadOutline" slot="icon-only" />
+          </ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content>
+      <div class="product-find">
+        <div class="product-search">
+          <ion-searchbar />
+        </div>
+        <div class="desktop-only">
+          <ion-list>
+            <ion-list-header>{{ $t("Catalog") }}</ion-list-header>
+            <ion-item>
+              <ion-label>{{ $t("Categories") }}</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">all</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-card>
+              <ion-card-header>
+                <ion-title>{{ $t("Purchase date") }}</ion-title>
+              </ion-card-header>
+
+              <ion-card-content>
+                <ion-chip>
+                  <ion-label>PO #</ion-label>
+                </ion-chip>
+                <ion-chip>
+                  <ion-label>PO #</ion-label>
+                </ion-chip>
+              </ion-card-content>
+            </ion-card>
+            <ion-item>
+              <ion-label>{{ $t("Size") }}</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">all</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-item>
+              <ion-label>{{ $t("Color") }}</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">all</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-card>
+              <ion-card-header>
+                <ion-title>{{ $t("Purchase date") }}</ion-title>
```

Comment:
Please refer to ionic framework documentation for using actual UI component.
https://ionicframework.com/docs/api/card

Reply:
ok